### ISSUE TITLE
LDAP support

### ISSUE THREAD

**Issue by tblancog** [laradock/laradock#710]
How can I enable **extension=php_ldap.dll** in **php.ini** file?
Tried to do it by entering in the workspace container and uncommenting that line but it didn't work.
Any ideas?

**Comment by cabrerabywaters** [comment]
@tblancog  Easiest way to do it is by using docker-php 
`docker-php-ext-configure ldap --with-libdir=lib/x86_64-linux-gnu/ && \
     docker-php-ext-install ldap`
should do the trick

I responded to your question also in [](https://github.com/laradock/laradock/pull/671)

Make sure to use those env variables to trigger the install ;)

**Comment by tblancog** [comment]
Thanks for the reply @cabrerabywaters. Where do I have to add that? Do I install docker-php? or inside laradock container?, sorry but i'm kind of newbie using docker and docker-compose

**Comment by cabrerabywaters** [comment]
@tblancog  No problem!...If you cannot wait until the pull request gets merge into the master branch..I would recommend just doing a quick change to the Dockerfile (laradock/php-fpm/Dockerfile-70) ...just forget about the env variables and paste the following code in line 17 after de maintainer statement.

``
RUN apt-get update && \
     apt-get install libldap2-dev -y && \
     docker-php-ext-configure ldap --with-libdir=lib/x86_64-linux-gnu/ && \
     docker-php-ext-install ldap 
``

Just like in this image...
<img width="959" alt="screenshot 2017-03-24 13 38 03" src="https://cloud.githubusercontent.com/assets/3051507/24304182/463dc26a-1097-11e7-8aba-5a1245e81864.png">

After you have done that..you need to rebuild your php-fpm image...that's an easy one..just go to your laradock root folder from your terminal and type:

``
docker-compose build php-fpm
``
That command will make sure that your php-fpm image now has the latest changes that we made.

after that you are ready to go! 
``
docker-compose up -d <whatever you need to use>
``

**Comment by tblancog** [comment]
Tried whtat  you said and I got this weird error when building the package..
```
Fetched 9878 kB in 13s (750 kB/s)
Reading package lists...
W: Size of file /var/lib/apt/lists/httpredir.debian.org_debian_dists_jessie-updates_main_binary-amd64_Packages.gz is not what the server reported 17571 17587
/bin/sh: 1:  apt-get: not found
ERROR: Service 'php-fpm' failed to build: The command '/bin/sh -c apt-get update && \ apt-get install libldap2-dev -y && \ docker-php-ext-configure ldap --with-libdir=lib/x86_64-linux-gnu/ && \ docker-php-ext-install ldap' returned a non-zero code: 127

```
I guess I'll wait for the PR to merge,
Thanks a lot @cabrerabywaters .

**Comment by cabrerabywaters** [comment]
@tblancog I just give it a try and actually received the same message.
<img width="1370" alt="screenshot 2017-03-24 17 11 32" src="https://cloud.githubusercontent.com/assets/3051507/24311993/5ab1d150-10b5-11e7-963d-de3bae648a8b.png">

but then the build continue with no problem... 
<img width="1323" alt="screenshot 2017-03-24 17 12 04" src="https://cloud.githubusercontent.com/assets/3051507/24312020/7aba6ffc-10b5-11e7-9ace-60854852080c.png">

And the image was built successfully

<img width="400" alt="screenshot 2017-03-24 17 16 12" src="https://cloud.githubusercontent.com/assets/3051507/24312062/9fa618de-10b5-11e7-8ff2-2d87e2601620.png">

**Comment by winfried-van-loon** [comment]
This issue should have been fixed in #933. 👍  
If you're still experiencing this issue, feel free to let us know by leaving a comment and we'll reopen the issue.

**Comment by raylucker** [comment]
> This issue should have been fixed in #933. 👍
> If you're still experiencing this issue, feel free to let us know by leaving a comment and we'll reopen the issue.

Hey, I'm still experiencing the issue. Now using PHP 7.1

**Comment by adamgdoura6** [comment]
yeah same here, any clue how to solve this? 
![image](https://github.com/user-attachments/assets/b0904c31-b2fb-46ea-bf47-5792fb766e43)
